src/init/features/frameworks/index.ts

```diff
@@ -0,0 +1,68 @@
+import * as clc from "colorette";
+import * as utils from "../../../utils";
+import { logger } from "../../../logger";
+import { promptOnce } from "../../../prompt";
+import {
+  DEFAULT_REGION,
+  ALLOWED_REGIONS,
+  DEFAULT_DEPLOY_METHOD,
+  ALLOWED_DEPLOY_METHODS,
+} from "./constants";
+import { Options } from "../../../options";
+import { ensure } from "../../../ensureApiEnabled";
+import { Config } from "../../../config";
+import { requirePermissions } from "../../../requirePermissions";
+import { isEnabled } from "../../../experiments";
+
+/**
+ * Setup new frameworks project.
+ */
+export async function doSetup(setup: any, config: Config, options: Options): Promise<void> {
+  const projectId = setup?.rcfile?.projects?.default;
+  if (projectId && !isEnabled("frameworks")) {
+    await requirePermissions({ ...options, project: projectId });
+    await Promise.all([ensure(projectId, "placeholder.googleapis.com", "unused", true)]);
```

Comment:
nit* we might want to comment this out for now? Does this actually work?

Reply:
The line to get the `projectId` works, but the we can remove `if` condition for now. The condition was using in other places.